web-frontend/src/mirage/models/index.ts

```diff
@@ -3,13 +3,13 @@ import { ModelDefinition } from 'miragejs/-types';
 
 import { AccountObject } from '../../models/AccountModel';
 import { TransferModel } from '../../models/TransferModel';
-import { IToken } from '@/models/AssetModel';
+import { Token } from '@/models/AssetModel';
```

Comment:
replace token by "Asset" to limit conflicts with https://github.com/massalabs/station-massa-wallet/pull/604